src/hardware/vga_memory.cpp

```diff
@@ -1071,13 +1069,8 @@ void VGA_SetupMemory(Section* sec) {
 	if (vga_allocsize<512*1024) vga_allocsize=512*1024;
 	// We reserve extra 2K for one scan line
 	vga_allocsize+=2048;
-	vga.mem.linear_orgptr = new uint8_t[vga_allocsize+16];
-	vga.mem.linear=(uint8_t*)(((Bitu)vga.mem.linear_orgptr + 16-1) & ~(16-1));
 	memset(vga.mem.linear,0,vga_allocsize);
 
-	vga.fastmem_orgptr = new uint8_t[(vga.vmemsize<<1)+4096+16];
-	vga.fastmem=(uint8_t*)(((Bitu)vga.fastmem_orgptr + 16-1) & ~(16-1));
-
```

Comment:
This is where the previous alignment happened; maybe we can add an assert to guarantee it:

``` c++
assert(reinterpret_cast<uintptr_t>(vga.fastmem) % vga_memalign == 0);
```